src/package.json

```diff
@@ -5,12 +5,12 @@
   "main": "fingerprint-auth",
   "nativescript": {
     "platforms": {
-      "android": "2.5.0",
-      "ios": "2.5.0"
+      "android": "3.0.0",
```

Comment:
Wouldn't this mean users MUST have {N} 3 in order to use the plugin? I'd rather keep this version as low as possible for that reason.

So - is there any reason why this plugin will no longer work with versions prior to {N} 3?